fbw-a380x/src/systems/instruments/src/MFD/shared/utils.ts

```diff
@@ -3,8 +3,8 @@
 
 import { Approach, ApproachType } from '@flybywiresim/fbw-sdk';
 
-export function getEtaFromUtcOrPresent(seconds: number | null, fromPresent: boolean) {
-  if (seconds === null) {
+export function getEtaFromUtcOrPresent(seconds: number | null | undefined, fromPresent: boolean) {
+  if (seconds === null || seconds === undefined) {
```

Comment:
```suggestion
  if (!seconds && seconds !== 0) {
```

May also be useful to check for `NaN` and `""`

Reply:
Or perhaps gracefully handle these cases and error log it as these should never be thrown.